### PR TITLE
resubmitting previous PR

### PR DESCRIPTION
**Pull request by jmay4826**

### CODE REVIEW

File: src/components/OneOnOnes/OneOnOnes.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false
+    };
+  }
+
+  componentDidMount() {
+    // this.props.getOneOnOnes(this.props.defaultCohort);
+  }
```
Comment (SheaClose) [inline src/components/OneOnOnes/OneOnOnes.js]: Unnecessary CDM

File: src/components/OneOnOnes/OneOnOnes.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  render() {
+    const students =
+      this.props.students.filter(
+        session => session.name === this.props.defaultCohort
+      )[0] || [{ classSession: [] }][0];
+    console.log(students);
```
Comment (SheaClose) [inline src/components/OneOnOnes/OneOnOnes.js]: Unnecessary console.log

File: src/components/Utils/SelectCohort.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      <MenuItem key={x.name} primaryText={x.name} value={x.name} />
+    ))}
+  </DropDownMenu>
+);
+const mapStateToProps = state => ({
+  cohorts: state.mainReducer.students,
+  selectedCohort: state.mainReducer.selectedCohort,
+  defaultCohort: state.mainReducer.defaultCohort
+});
+
```
Comment (SheaClose) [inline src/components/Utils/SelectCohort.js]: maybe something like this?

```
function mapStateToProps(state) {
  const {
    students: cohorts,
    selectedCohort,
    defaultCohort
  } = state.mainReducer;
  return {
    cohorts,
    selectedCohort,
    defaultCohort
  };
}
```

**Comment by jmay4826** [comment]
made edits

**Pull request merged**